js/hoverzoom.js

```diff
@@ -1678,6 +1678,14 @@ var hoverZoom = {
                     saveImage();
                     return false;
                 }
+                if (event.which == options.copyImageKey) {
```

Comment:
I think it's better to put an `if (isChromiumBased)` block around these two statements to avoid processing these shortcuts on Firefox while not showing the options to re-assign and disable them.

Reply:
Good catch. This could have definitely prevented "C" from being assigned to a shortcut processed later. Fixed.